LabExT/Movement/MoverNew.py

```diff
@@ -504,7 +504,8 @@ def move_absolute(
         movement_commands: Dict[Orientation, Type[ChipCoordinate]],
         chip: Type[Chip],
         wait_for_stopping: bool = True,
-        wait_timeout: float = 2.0
+        wait_timeout: float = 2.0,
+        stages_are_lifted: bool = False
```

Comment:
Replace this flag with a selector: `move_stages_lifted: bool = True`.

This flag should select if the stages should be lifted-moved-lowered or just moved.